dts/arm/nuvoton/npcx/npcx7-psl-ctrl-map.dtsi

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	def_psl_conf_list {
+		compatible = "nuvoton,npcx-pslctrl-conf";
+
+		/* PSL input IO configurations */
+		psl_in1: psl_iod2 {
```

Comment:
psi-iod2

Please check that your node names use hyphens. Only phandles should use underscore.

Also check the commit message above

Reply:
Done. Thanks for reminding. Regarding the naming rule for nodes and phandles, I will submit the other PR to review all device-tree files of npcx7 series.